docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.md

```diff
@@ -32,71 +32,68 @@ verdi daemon start
 
 ``` python
- code = Code.get_from_string(<codename>)
+code = load_code("<codename>")
 ```
 
-Here `Code` is the general AiiDA class handling all possible codes, and `code` is a class instance tagged as `<codename>` (see the first part of the tutorial for listing all codes installed in your AiiDA machine). You might also want to list only the codes that define a default calculation plugin for the pw.x code of Quantum ESPRESSO. You can do this with the following command:
+Here `load_code` (imported for you by `verdi shell` or the `verdi run` command) returns a `Code` which is the general AiiDA class handling all possible codes, and `code` is an instance labbeled as `<codename>` (see the first part of the tutorial for listing all codes installed in your AiiDA machine). You might also want to list only the codes that define a default calculation plugin for the pw.x code of Quantum ESPRESSO. You can do this with the following command: 
 
 ```console
-verdi code list -p quantumespresso.pw
+verdi code list -P quantumespresso.pw
 ```
 
-Pick the correct codename, that might look like, e.g. `qe-pw-6.2.1@localhost`.
+Pick the correct codename, that might look like, e.g. `qe-6.3-pw@localhost`.
 
-Once run, AiiDA calculations are instances of the class `Calculation`, more precisely of one of its subclasses, each corresponding to a code specific plugin (for example, the PWscf plugin). You have already seen `Calculation` classes in the previous sections.
+Once run, AiiDA calculations are instances of the class `CalcJob`, more precisely of one of its subclasses, each corresponding to a code specific plugin (for example, the PWscf plugin). You have already seen `CalcJob` classes in the previous sections.
 
 However, to create a new calculation, rather than manually creating a new class, the suggested way is to use a `Builder`, that helps in setting the various calculation inputs and parameters, and provides TAB-completion.
 
 To obtain a new builder, we can use the `get_builder` method of the `code` object:
 
 ``` python
- builder = code.get_builder()
+builder = code.get_builder()
 ```
 
-This returns a builder that helps in setting up the inputs for the `PwCalculation` class (associated to the `quantumespresso.pw` plugin, i.e. the default plugin for the code you chose before).
+This returns a builder that helps in setting up the inputs for the `PwCalculation` class (associated to the `quantumespresso.pw` plugin, the default plugin for the code you chose before).
 
 As the first step, you can assign a (short) label or a (long) description to the calculation that you are going to create, that you might find convenient in the future. This can be achieved with:
 
 ``` python
- builder.label = "PW test"
- builder.description = "My first AiiDA calc with Quantum ESPRESSO on BaTiO3"
+builder.metadata.label = "PW test"
+builder.metadata.description = "My first AiiDA calc with Quantum ESPRESSO on Si"
```

Comment:
just wondering whether the calculation they are doing is on Si or BaTiO3 here...

Reply:
I'll double check tomorrow, I'm not sure myself.

Reply:
Indeed, at that point users are working with an `Si` structure.